lib/playlistGenerator/PlaylistGenerator.js

```diff
@@ -94,6 +118,7 @@ var setPlayList = function (playlist){
         that.playlistImp.inner.liveWindowDuration = Math.ceil(that.playListLimits.manifestTimeWindowInMsec / hlsStoreFilefactor);
     }
     that.playlistImp.addListener(that);
+    that.playlistImp.inner.presentationEndTime = 0
```

Comment:
if(live) 
that.playlistImp.inner.presentationEndTime = that.playlistImp.getCurrentTime() + MILLISEC_IN_YEAR * 10;
else 
that.playlistImp.inner.presentationEndTime = 0